lib/ssm_storage/db.rb

```diff
@@ -31,6 +31,20 @@ def constant_exists?
       Object.const_defined? ACTIVE_RECORD_MODEL
     end
 
+    def convert_boolean(value)
+      value = value.to_s.downcase
+      return true if value[0] == 't'
+      return false if value[0] == 'f'
+      raise 'Not a valid boolean: must be one of true or false'
```

Comment:
@lorman do we want to restrict false if the string starts with `f` on the value portion ?

Reply:
I see no reason not to. As long as the type is set to `b` we should be safe.